string robot_name
uint8 id
float32[] pose
float32[] descriptor
float32[] navigableDirection
BinaryArrayMsg[] frontierPoints
float32[] frontierDistance